client/blocks/inline-help/admin-sections.js

```diff
@@ -444,6 +443,6 @@ export function filterListBySearchTerm( searchTerm = '', collection = [], limit
 	} );
 
 	return [ ...exactMatches, ...partialMatches, ...synonymMatches ]
-		.map( ( item ) => ( { ...item, support_type: SUPPORT_TYPE_ADMIN_SECTION, key: item.title } ) )
+		.map( ( item ) => ( { ...item, key: item.title } ) )
```

Comment:
This `.map` could be removed completely. Every "admin section" result already has a `link` field, and since the `.link` field is preferred over `.key` when rendering, the `key` field will never be used.

Reply:
I need to rebase this branch, this change should already be in `trunk`.